src/led_ble/led_ble.py

```diff
@@ -622,13 +665,23 @@ def _resolve_characteristics(self, services: BleakGATTServiceCollection) -> bool
             if char := services.get_characteristic(characteristic):
                 self._write_char = char
                 break
+        _LOGGER.debug(
+            "using characteristic %s for read, characteristic %s for write",
+            self._read_char,
+            self._write_char,
+        )
         return bool(self._read_char and self._write_char)
 
     async def _resolve_protocol(self) -> None:
         """Resolve protocol."""
         if self._resolve_protocol_event.is_set():
             return
-        await self._send_command_while_connected([STATE_COMMAND])
+        if self._is_hello_fairy():
+            await self._send_command_while_connected(
+                [b"\xaa\x00\x00\xaa"]
+            )  # get version and capabilities
+        else:
+            await self._send_command_while_connected([STATE_COMMAND])
```

Comment:
Please add a new method to the protocol to get the state command instead

Reply:
that method is implemented but the protocol is not resolved yet at this point